samples/drivers/misc/ft800/sample.yaml

```diff
@@ -2,6 +2,7 @@ sample:
   name: FT800 display sample
 tests:
   sample.display.ft800:
+    build_only: true
```

Comment:
we should avoid build_only here and instead set a harness, i.e. 

```
   harness: display
```

Reply:
Ok. So if it is testable I also added     
```
harness_config:
      fixture: spi_display_ft800
```
as the setup requires a fixture i.e. display to run